src/parsers/trades/index.ts

```diff
@@ -14,6 +15,8 @@ const parserMapping: {[key in EXCHANGES]: any} = {
     [EXCHANGES.Kraken]: krakenParser,
     [EXCHANGES.Poloniex]: poloniexParser,
     [EXCHANGES.Revolut]: revolutParser,
+    [EXCHANGES.Pionex]: pionexParser,
```

Comment:
Dust parser is for the dust collector feature in Pionex, it is in a separate csv file

Reply:
With a different format, hence the different header

Reply:
It is an exchange specialized in trading bots, with mostly trades and transactions.

Reply:
Yeah but what is this dust collector? Are they actually trades too?

I think it might be better(assuming these are trades), to change the format so each exchange supports an array of hashes to match against and then in your parser you can check to see if its dust or not and then process based on that. So users dont need to choose

Reply:
"Dust Collector is a feature on Pionex exchange that helps you collect all the small left behind assets into a whole. For example, when you convert btc to usdt there are a little fraction that is left behind. The dust collector helps you collect them."

I have seen that on other exchanges too. I think legally it must be considered a trade, at least in my country.
It allows converting very small amount of crypto into USDT usually when the amount is too small to do a normal trade.

Another use is when there are many different coins left, even if the amount is not that small, it will convert all into one coin.

I like your approach, so I will try to implement that testing against an array.

Reply:
But that will mess with my other PRs. Let’s try to merge the other ones first.

Reply:
I included a way to support multiple hashes which is very little breaking, so that it will be easy to merge